presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/BaseJdbcClient.java

```diff
@@ -193,8 +195,11 @@ public BaseJdbcClient(
         }
     }
 
-    protected boolean filterSchema(String schemaName)
+    protected boolean filterSchema(@Nullable String schemaName)
```

Comment:
Is there an issue it fixes? What is the case when null is passed here?

Reply:
@losipiuk Phoenix, see @lhofhansl 's https://prestosql.slack.com/archives/CP1MUNEUX/p1595558846274400 

@ebyhr 
When a connector reports some tables with null schema, there are inaccessible
in current base jdbc implementation. To properly handle them you'd need to change
more places besides this one. Thus, I would consider null schemas a concrete
connnector responsibility until we determine how frequent or how problematic this
approach is.

Reply:
quoting @lhofhansl 
> In Phoenix tables do not need to have a schema. For that the connector adds a virtual "default" schema.

this means it's not OK to filter them out.
perhaps the Phoenix code could provide `firstNonNull(schemaname, "default")` here.

Also, this probably means we should be able to test those null schemas.

Reply:
Thanks for fixing this so quickly (I offered to provide a fix :) ).
Phoenix can actually be run completely without schemas - mostly for historical reasons.
Tables are allowed to have complex names (including . in that case).